Sources/SwiftDocC/Infrastructure/DocumentationContext.swift

```diff
@@ -2818,7 +2818,7 @@ public class DocumentationContext: DocumentationContextDataProviderDelegate {
 
     // MARK: - Relationship queries
     
-    /// Fetch the child nodes of a documentation node with the given `reference``,  optionally filtering to only children of the given `kind`.
+    /// Fetch the child nodes of a documentation node with the given `reference``, optionally filtering to only children of the given `kind`.
```

Comment:
There's also an unmatched number of \` around "reference".
```suggestion
    /// Fetch the child nodes of a documentation node with the given `reference`, optionally filtering to only children of the given `kind`.
```